src/OrbitGl/GlSlider.cpp

```diff
@@ -309,9 +305,9 @@ void GlHorizontalSlider::DoDraw(PrimitiveAssembler& primitive_assembler,
   // Triangle is 3 pixels smaller than the area - there is a 2 pixel border around the scrollbar,
   // and there is no margin between the border and the triangle to make it as big as possible.
   // On the other side, there is a 1 pixel margin between the triangle and the vertical line
-  float tri_size = slider_resize_pixel_margin_ - 3.f;
-  tri_size = std::min(tri_size, pixel_height_ - tri_size * kHeightFactor - 2.f);
-  const float tri_y_offset = (pixel_height_ - tri_size * kHeightFactor) / 2.f;
+  float tri_size = GetSliderResizeMargin() - 3.f;
```

Comment:
nit. with the renaming, we lose information about the units. At other places in the file, "pixels" is used to specify units. I guess everything is in pixels now, so we might want to make the naming consistent and remove "pixels" everywhere, but this is outside the scope of this PR.

Reply:
Yes, I would rather avoid using pixels in CaptureViewElements where we are using world_coordinates instead of Screen's ones, so I agree about removing pixels in this class.